GeoLib/AnalyticalGeometry.cpp

```diff
@@ -133,11 +133,16 @@ bool lineSegmentIntersect(
 	if (parallel(v,w)) {
 		if (parallel(pq,v)) {
 			// check if c is located at v (c-a = t (b-a), t in [0,1])
-			if (qp[0] / v[0] <= 1.0)
+			if (0 <= qp[0] / v[0] && qp[0] / v[0] <= 1.0) {
```

Comment:
> Can v[0] be 0.0?

Yes, this could be the case. This was a serious problem. I hopefully fixed it in commit 6e75c9e?! Please have a look.